satpy/scene.py

```diff
@@ -665,7 +665,7 @@ def aggregate(self, dataset_ids=None, boundary='exact', side='left', func='mean'
             except AttributeError:
                 resolution = max(target_area.lats.resolution, target_area.lons.resolution)
             for ds_id in ds_ids:
-                res = self[ds_id].coarsen(boundary=boundary, side=side, func=func, **dim_kwargs)
+                res = self[ds_id].coarsen(boundary=boundary, side=side, **dim_kwargs)
```

Comment:
Don't you need to pass the `func` positional argument?

Reply:
no, it isn't needed. It was actually `coord_func` and I don't think it needs to be the same as `func`, so I'm dropping it.

Reply:
Ah I didn't see it is used below.